components/tools/src/main/java/edu/jhuapl/dorset/components/tools/CommandLineClient.java

```diff
@@ -0,0 +1,104 @@
+/**
+ * 
+ */
+package edu.jhuapl.dorset.components.tools;
```

Comment:
That would make it easier for new developers to contribute. I think it is a good idea.